fmriprep/cli/parser.py

```diff
@@ -313,19 +334,32 @@ def _slice_time_ref(value, parser):
     )
     g_conf.add_argument(
         "--bold2t1w-init",
-        action="store",
-        default="register",
+        action=DeprecatedAction,
         choices=["register", "header"],
-        help='Either "register" (the default) to initialize volumes at center or "header"'
-        " to use the header information when coregistering BOLD to T1w images.",
+        help="Deprecated - use `--bold2anat-init` instead.",
     )
     g_conf.add_argument(
         "--bold2t1w-dof",
+        action=DeprecatedAction,
+        choices=[6, 9, 12],
+        type=int,
+        help="Deprecated - use `--bold2anat-dof` instead.",
+    )
+    g_conf.add_argument(
+        "--bold2anat-init",
+        choices=["auto", "t1w", "header"],
+        default="auto",
+        help="Method of BOLD to anatomical coregistration. If `auto`, a T2w image is used if "
+        "available, otherwise the T1w image. `t1w` forces use of the T1w, and `header` uses "
+        "the T1w header information.",
```

Comment:
```suggestion
        help="Method of initial BOLD to anatomical coregistration. If `auto`, a T2w image is used if "
        "available, otherwise the T1w image. `t1w` forces use of the T1w, and `header` uses "
        "the BOLD header information without an initial registration.",
```